src/profile-logic/process-profile.js

```diff
@@ -812,6 +812,8 @@ export function processProfile(
     toolkit: geckoProfile.meta.toolkit,
     version: geckoProfile.meta.version,
     preprocessedProfileVersion: CURRENT_VERSION,
+    appBuildID: geckoProfile.meta.appBuildID,
+    sourceURL: geckoProfile.meta.sourceURL,
```

Comment:
I see "undefined" for `sourceURL` when I try it, with latest nightly, is that expected?
Actually, logging `geckoProfile.meta` I don't see it, when is it supposed to be set? Can you add a comment with this information if that's happening only in some cases? (the comment should rather go on the 2 types than here).

Reply:
Hmm, this would mean that Gecko is not setting it, which would be a bug in my patch for bug 1451853. I'll investigate, but this changeset shouldn't be related as far as I can see.

Reply:
This is not urgent either, so let's investigate first. I'm mostly worried about the casing, as sometimes we write `URL` and sometimes `Url` :) (but this is not the case here as I couldn't find any occurrence of `sourceURL` at all, with any case).

Also it's not completely clear to me what `sourceURL` actually means, so a comment would be nice anyway, for the reader. Is it the link to the hg revision for this build?